ports/raspberrypi/boards/sparkfun_micromod_rp2040/pins.c

```diff
@@ -0,0 +1,105 @@
+#include "shared-bindings/board/__init__.h"
+
+STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
+    // D (Digital only) pins (D0,D1)
+    { MP_ROM_QSTR(MP_QSTR_D0), MP_ROM_PTR(&pin_GPIO7) },        // GPIO6 - D0
```

Comment:
This calls out GPI07 twice (Comment looks like it should be GPIO6)

Reply:
@zapwizard Thanks for catching that. I've created #4665 to address it.

Not sure that should block connecting with SPI, though. Are you using the default definition? I'll take a look an make sure that's correct.